### ISSUE TITLE
Header in stack navigator modal

### ISSUE THREAD

**Issue by donovanhiland** [react-navigation/react-navigation#7167]
Hi there! Thanks for all the awesome work that's being done here. I wasn't sure if here or the main react-navigation repo was a better place to post this so I'll start here and move if necessary.

I'm migrating a project away from react-native-navigation to react-navigation. I'm curious if this is still the best way to attach the maintained react-navigation stack navigator header to a modal view: https://medium.com/4thought-studios/better-modals-in-react-navigation-56c649e0249

TLDR:
Import the header from `react-navigation-stack/src/views/Header/Header.js` and manually render it at the top of the modal screens.

Is there a better or more supported way to achieve this?
